src/components/Upper.js

```diff
@@ -1,17 +1,31 @@
-import React from 'react';
+/* eslint-disable react/no-typos */
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.